ludwig/encoders/sequence_encoders.py

```diff
@@ -2040,6 +2113,9 @@ def __init__(
 
         logger.debug('  TransformerStack')
         self.transformer_stack = TransformerStack(
+            # todo: confirm that input_size should be hidden_size
```

Comment:
that is correct, because we are either embedding to that size or projecting to that size in the previous if-else

Reply:
While reviewing this comment, noticed that sequence encoders was not updated to handle `should_embed=False`.  Made changes to support this and updated unit test to test this parameter.  Expanded unit test cover 1,008 test cases.  Runs in about 10 seconds on my laptop.